service/src/main/java/com/commercetools/pspadapter/payone/transaction/TransactionBaseExecutor.java

```diff
@@ -0,0 +1,58 @@
+package com.commercetools.pspadapter.payone.transaction;
+
+import com.google.common.cache.LoadingCache;
+import io.sphere.sdk.payments.commands.updateactions.SetStatusInterfaceCode;
+import io.sphere.sdk.payments.commands.updateactions.SetStatusInterfaceText;
+import io.sphere.sdk.types.Type;
+
+import java.util.Map;
+
+/**
+ * @author mht@dotsource.de
+ * Common base class responsible for default paymentupdateactions
+ */
+public abstract class TransactionBaseExecutor extends IdempotentTransactionExecutor{
+
+    private final static String STATUS = "status";
+    private final static String ERROR_CODE = "errorcode";
+    private final static String CUSTOMER_MESSAGE = "customermessage";
```

Comment:
Created, but never used. Or `PaymentStatus` should be expanded, or `CustomField` should be created.